keyboards/handwired/slash/info.json

```
@@ -0,0 +1,74 @@
+{
+    "keyboard_name": "slash",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 15,
```

This change and the one below will make QMK Configurator render the keyboard the way it actually looks when built. Just edits dimensions and key coordinates.

```suggestion
    "width": 16.75,
```